src/core/js/helpers.js

```diff
@@ -355,6 +355,18 @@ define([
         a11y_attrs_tabbable: function() {
             Adapt.a11y.log.deprecated('a11y_attrs_tabbable should not be used. tabbable elements should be natively tabbable.');
             return new Handlebars.SafeString(' role="region" tabindex="0" ');
+        },
+
+        /**
+         * Produce display text with alternative screen reader version.
+         * @param {string} visible Text that will be displayed on screen
+         * @param {string} alternatives Text that will be read by the screen reader (instead of what's displayed on screen)
+         * @usage {{a11y_alt_text '$5bn' 'five billion dollars'}} or {{a11y_alt_text 'Here are some bits to read' 'There are' _items.length 'items to read'}}
```

Comment:
sorry totally my mistake
```suggestion
         * @example {{a11y_alt_text '$5bn' 'five billion dollars'}} or {{a11y_alt_text 'Here are some bits to read' 'There are' _items.length 'items to read'}}
```